src/App.tsx

```diff
@@ -1,61 +1,34 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm';
+import { findUserById } from './utils/FindUserById';
+import { Todo } from './types/Todo';
+
+import todosFromServer from './api/todos';
+
+const preparedTodos: Todo[] = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: findUserById(todo.userId),
+  };
+});
+
+preparedTodos.sort((todo1, todo2) => todo1.id - todo2.id);
```

Comment:
[CODE KNOWLEDGE] - You are using the sort method on the array directly which is a mutating method. This can lead to bugs and unpredictable behavior in your code. It's better to create a copy of the array before sorting it. Here is a suggestion:jsxconst sortedTodos = [...preparedTodos].sort((todo1, todo2) => todo1.id - todo2.id);```